### PR TITLE
fix: update the script to build/verify docs with loopback.io

### PR DESCRIPTION
**Pull request by raymondfeng**
The previous script rely on `packages/build` to clean up
`loopback.io` directory after verification but it does not
install deps of `@loopback/build`.

See https://travis-ci.org/strongloop/loopback-next/jobs/369943689

## Checklist

- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

### CODE REVIEW

File: bin/verify-docs.sh [hunk 1]
```diff
@@ -1,17 +0,0 @@
...
-set -ev
-
-# Make sure we use the correct repo root dir
-DIR=`dirname $0`
-REPO_ROOT=$DIR/..
-pushd $REPO_ROOT >/dev/null
-rm -rf sandbox/loopback.io/
-# Shadow clone is faster
-git clone --depth 1 https://github.com/strongloop/loopback.io.git sandbox/loopback.io
-lerna bootstrap --scope loopback.io-workflow-scripts
```
Comment (virkt25) [inline bin/verify-docs.sh]: Thanks for the fix @raymondfeng ! A simpler fix would be to just add `@loopback/build` in the scope so it's bootstrapped. The following worked for me. 

`lerna bootstrap --scope loopback.io-workflow-scripts --scope @loopback/build --scope @loopback/docs`

Comment (raymondfeng) [reply bin/verify-docs.sh]: I would rather have all the steps in the shell script itself. `@loopback/build` has a lot of dependencies and it's time consuming to bootstrap it. Please note the PR also adds a lot of comments in the script.

File: package.json [hunk 2]
```diff
@@ -40,8 +40,8 @@
     "build:full": "npm run clean:lerna && npm run bootstrap && npm test",
     "pretest": "npm run clean && npm run build",
     "test": "node packages/build/bin/run-nyc npm run mocha --scripts-prepend-node-path",
-    "verify:docs": "npm run build:site && node packages/build/bin/run-clean \"sandbox/loopback.io\"",
```
Comment (b-admike) [inline package.json]: I guess this was the main cause for the issue seen in https://travis-ci.org/strongloop/loopback-next/jobs/369943689?

File: bin/verify-docs.sh (hunk 1 above)
Comment (b-admike) [reply bin/verify-docs.sh]: +1 for having them in the shell script if that is the case.

File: package.json (hunk 2 above)
Comment (raymondfeng) [reply package.json]: Yes

File: bin/verify-docs.sh (hunk 1 above)
Comment (virkt25) [reply bin/verify-docs.sh]: Fair enough! And yay for the comments!

**Pull request merged**